contrib/experimental/great_expectations_experimental/rule_based_profiler/data_assistant_result/statistics_data_assistant_result.py

```diff
@@ -0,0 +1,37 @@
+from typing import Dict, Tuple, Union
+
+from great_expectations.rule_based_profiler.altair import AltairDataTypes
+from great_expectations.rule_based_profiler.data_assistant_result import (
+    DataAssistantResult,
+)
+
+
+class StatisticsDataAssistantResult(DataAssistantResult):
+    @property
+    def metric_expectation_map(self) -> Dict[Union[str, Tuple[str]], str]:
+        """
+        A mapping is defined for which metrics to plot and their associated expectations.
```

Comment:
could add a note saying plotting is not within scope?

Reply:
@Shinnnyshinshin This has been done.  Thanks!